js/components/generate/__tests__/generate.test.js

```diff
@@ -34,6 +34,10 @@ describe("Handle invalid course inputs appropriately", () => {
     },
   ])(".$coursesInputText", async ({ coursesInputText, expectedWarning }) => {
     const user = userEvent.setup()
+
+    const categorySelect = screen.getByRole("combobox")
```

Comment:
See my above comment in the `GenerateForm` component about making `courses` the default.